src/lib/utilities/get-namespace.ts

```diff
@@ -15,13 +15,12 @@ export const getNamespace = ({
   namespace,
   defaultNamespace,
   namespaces,
-}: GetNamespaceParameters): string => {
-  if (
-    namespace &&
-    namespaces &&
-    namespaces.find((ns) => ns.namespaceInfo.name === namespace)
-  ) {
-    return namespace;
+}: GetNamespaceParameters): string | undefined => {
+  if (namespace && namespaces) {
```

Comment:
What about…

```js
if (!namespace) return;
if (!namespaces) return;
return namespaces.find((ns) => ns.namespaceInfo.name === namespace);
```

Reply:
That way it's easier to reason about the steps we take to make sure we should do the thing.